web/src/main/java/org/openmrs/web/filter/StartupFilter.java

```diff
@@ -196,6 +195,11 @@ private void initializeVelocity() {
 		}
 	}
 	
+	private Runtime Runtime() {
+		// TODO Auto-generated method stub
```

Comment:
Is this change related to updating apache velocity?

Reply:
@dkayiwa ,,it was as a result of fixing a compilation error that occurred during the maven build after updating apache velocity

Reply:
hello @dkayiwa the above change was due to the updating of apache velocity from 1.6 to 2.0 velocity core engine